### PR TITLE
Frontend/development/CSS-Files-Migration

### PR DESCRIPTION
**Pull request by AliNasra**
I have removed the CSS code from the jsx files of Sign-in and Sign-up pages. Nevertheless, there are a lot of errors that prevent the code from compiling properly.

### CODE REVIEW

**Comment by AliNasra** [comment]
I have rectified the problems in general though there is a little anomaly with the Sign-Up page's UI,

**Comment by enfurars** [comment]
What is the anomaly?

File: prediction-polls/frontend/src/Pages/Auth/SignIn/SignIn.module.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  color: var(--primary-700);
+  border-color: var(--primary-700);
+}
+
+.imageStyle {
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: linear-gradient(var(--primary-900), var(--primary-800));
```
Comment (selinisik) [inline prediction-polls/frontend/src/Pages/Auth/SignIn/SignIn.module.css:32]: Sign-in page background color is not correct.

**Review by selinisik — CHANGES_REQUESTED** [review]

**Review by enfurars — CHANGES_REQUESTED** [review]
Button colors of the sign up page are not correct. Padding can be checked also.

**Comment by selinisik** [comment]
Due to urgency, I fixed the errors in the PR!

**Pull request merged**